### PR TITLE
Aug21 update; xattr -c cleanup

### PR DESCRIPTION
**Pull request by durack1**
Ref #257 

@mauzey1 please review and merge

### CODE REVIEW

File: staff/durack/publication.html [hunk 1]
```diff
@@ -119,20 +118,27 @@ <h2>Publications:</h2>
 <p>
 <b>Book Chapters:</b><br>
 <br>
-
+Douville, H., K. Raghavan, J. Renwick, R. P. Allan, P. A. Arias, M. Barlow, R. Cerezo-Mota, A. Cherchi, T. Y. Gan, J. Gergis, D. Jiang, A. Khan, W. Pokam Mba, D. Rosenfield, J. Tierney, O. Zolina and <b>contributing authors</b> (2021): Water cycles changes (Chapter 8). In Climate Change 2021: The Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change [Masson-Delmotte, V., P. Zhai, A. Pirani, S. L. Connors, C. Pean, S. Berger, N. Caud, Y. Chen, L. Goldfarb, M. I. Gomis, M. Huang, K. Leitzell, E. Lonnoy, J. B. R. Matthews, T. K. Maycock, T. Waterfield, O. Yelekçi, R. Yu and B. Zhou (eds.)]. Cambridge University Press. <b>in press</b>. Cambridge University Press, Cambridge, United Kingdom and New York, NY, USA. 239 pp. DOI: xxx. <a href="links/AR6Chap08.html" target="_blank">Accessible online</a>
```
Comment (mauzey1) [inline staff/durack/publication.html:121]:
```suggestion
Douville, H., K. Raghavan, J. Renwick, R. P. Allan, P. A. Arias, M. Barlow, R. Cerezo-Mota, A. Cherchi, T. Y. Gan, J. Gergis, D. Jiang, A. Khan, W. Pokam Mba, D. Rosenfield, J. Tierney, O. Zolina and <b>contributing authors</b> (2021): Water cycles changes (Chapter 8). In Climate Change 2021: The Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change [Masson-Delmotte, V., P. Zhai, A. Pirani, S. L. Connors, C. Pean, S. Berger, N. Caud, Y. Chen, L. Goldfarb, M. I. Gomis, M. Huang, K. Leitzell, E. Lonnoy, J. B. R. Matthews, T. K. Maycock, T. Waterfield, O. Yelekçi, R. Yu and B. Zhou (eds.)]. Cambridge University Press. <b>in press</b>. Cambridge University Press, Cambridge, United Kingdom and New York, NY, USA. 239 pp. DOI: xxx. <a href="links/AR6Chap08.html" target="_blank">Accessible online</a><br>
```

File: staff/durack/publication.html [hunk 2]
```diff
@@ -119,20 +118,27 @@ <h2>Publications:</h2>
...
+<br>
+Eyring, V., N. P. Gillett, K. M. AchutaRao, R. Barimalala, M. Barreiro Parillo, N. Bellouin, C. Cassou, <b>P. J. Durack</b>, Y. Kosaka, S. McGregor, S.-K. Min, O. Morgenstern, Y. Sun and contributing authors (2021): Human influence on the climate system (Chapter 3). In Climate Change 2021: The Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change [Masson-Delmotte, V., P. Zhai, A. Pirani, S. L. Connors, C. Pean, S. Berger, N. Caud, Y. Chen, L. Goldfarb, M. I. Gomis, M. Huang, K. Leitzell, E. Lonnoy, J. B. R. Matthews, T. K. Maycock, T. Waterfield, O. Yelekçi, R. Yu and B. Zhou (eds.)]. Cambridge University Press. <b>in press</b>. Cambridge University Press, Cambridge, United Kingdom and New York, NY, USA. 207 pp. DOI: xxx. <a href="links/AR6Chap03.html" target="_blank">Accessible online</a>
```
Comment (mauzey1) [inline staff/durack/publication.html:123]:
```suggestion
Eyring, V., N. P. Gillett, K. M. AchutaRao, R. Barimalala, M. Barreiro Parillo, N. Bellouin, C. Cassou, <b>P. J. Durack</b>, Y. Kosaka, S. McGregor, S.-K. Min, O. Morgenstern, Y. Sun and contributing authors (2021): Human influence on the climate system (Chapter 3). In Climate Change 2021: The Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change [Masson-Delmotte, V., P. Zhai, A. Pirani, S. L. Connors, C. Pean, S. Berger, N. Caud, Y. Chen, L. Goldfarb, M. I. Gomis, M. Huang, K. Leitzell, E. Lonnoy, J. B. R. Matthews, T. K. Maycock, T. Waterfield, O. Yelekçi, R. Yu and B. Zhou (eds.)]. Cambridge University Press. <b>in press</b>. Cambridge University Press, Cambridge, United Kingdom and New York, NY, USA. 207 pp. DOI: xxx. <a href="links/AR6Chap03.html" target="_blank">Accessible online</a><br>
```

File: staff/durack/publication.html [hunk 3]
```diff
@@ -119,20 +118,27 @@ <h2>Publications:</h2>
...
+<br>
+Fox-Kemper, B., H. T. Hewitt, C. Xiao, G. Adalgeirsdottir, S. Drijfhout, T. L. Edwards, N. R. Golledge, M. Hemer, R. E. Kopp, G. Krinner, A. Mix, D. Notz, S. Nowicki, I. S. Nurhati, L. Ruiz, J.-B. Sallee, A. B. A. Slangen, Y. Yu and <b>contributing authors</b> (2021): Ocean, cryosphere and sea level change (Chapter 9). In Climate Change 2021: The Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change [Masson-Delmotte, V., P. Zhai, A. Pirani, S. L. Connors, C. Pean, S. Berger, N. Caud, Y. Chen, L. Goldfarb, M. I. Gomis, M. Huang, K. Leitzell, E. Lonnoy, J. B. R. Matthews, T. K. Maycock, T. Waterfield, O. Yelekçi, R. Yu and B. Zhou (eds.)]. Cambridge University Press. <b>in press</b>. Cambridge University Press, Cambridge, United Kingdom and New York, NY, USA. 271 pp. DOI: xxx. <a href="links/AR6Chap09.html" target="_blank">Accessible online</a>
```
Comment (mauzey1) [inline staff/durack/publication.html:125]:
```suggestion
Fox-Kemper, B., H. T. Hewitt, C. Xiao, G. Adalgeirsdottir, S. Drijfhout, T. L. Edwards, N. R. Golledge, M. Hemer, R. E. Kopp, G. Krinner, A. Mix, D. Notz, S. Nowicki, I. S. Nurhati, L. Ruiz, J.-B. Sallee, A. B. A. Slangen, Y. Yu and <b>contributing authors</b> (2021): Ocean, cryosphere and sea level change (Chapter 9). In Climate Change 2021: The Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change [Masson-Delmotte, V., P. Zhai, A. Pirani, S. L. Connors, C. Pean, S. Berger, N. Caud, Y. Chen, L. Goldfarb, M. I. Gomis, M. Huang, K. Leitzell, E. Lonnoy, J. B. R. Matthews, T. K. Maycock, T. Waterfield, O. Yelekçi, R. Yu and B. Zhou (eds.)]. Cambridge University Press. <b>in press</b>. Cambridge University Press, Cambridge, United Kingdom and New York, NY, USA. 271 pp. DOI: xxx. <a href="links/AR6Chap09.html" target="_blank">Accessible online</a><br>
```

File: staff/durack/publication.html [hunk 4]
```diff
@@ -119,20 +118,27 @@ <h2>Publications:</h2>
...
 <br>
+Gulev, S. K., P. W. Thorne, J. Ahn, F. J. Dentener, C. M. Domingues, S. Gerland, D. Gong, D. S. Kaufman, H. C. Nnamchi, J. Quaas, J. A. Rivera, S. Sathyendranath, S. L. Smith, B. Trewin, K. von Shuckmann, Russell S. Vose and <b>contributing authors</b> (2021): Changing state of the climate system (Chapter 2). In Climate Change 2021: The Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change [Masson-Delmotte, V., P. Zhai, A. Pirani, S. L. Connors, C. Pean, S. Berger, N. Caud, Y. Chen, L. Goldfarb, M. I. Gomis, M. Huang, K. Leitzell, E. Lonnoy, J. B. R. Matthews, T. K. Maycock, T. Waterfield, O. Yelekçi, R. Yu and B. Zhou (eds.)]. Cambridge University Press. <b>in press</b>. Cambridge University Press, Cambridge, United Kingdom and New York, NY, USA. 223 pp. DOI: xxx. <a href="links/AR6Chap02.html" target="_blank">Accessible online</a>
```
Comment (mauzey1) [inline staff/durack/publication.html:129]:
```suggestion
Gulev, S. K., P. W. Thorne, J. Ahn, F. J. Dentener, C. M. Domingues, S. Gerland, D. Gong, D. S. Kaufman, H. C. Nnamchi, J. Quaas, J. A. Rivera, S. Sathyendranath, S. L. Smith, B. Trewin, K. von Shuckmann, Russell S. Vose and <b>contributing authors</b> (2021): Changing state of the climate system (Chapter 2). In Climate Change 2021: The Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change [Masson-Delmotte, V., P. Zhai, A. Pirani, S. L. Connors, C. Pean, S. Berger, N. Caud, Y. Chen, L. Goldfarb, M. I. Gomis, M. Huang, K. Leitzell, E. Lonnoy, J. B. R. Matthews, T. K. Maycock, T. Waterfield, O. Yelekçi, R. Yu and B. Zhou (eds.)]. Cambridge University Press. <b>in press</b>. Cambridge University Press, Cambridge, United Kingdom and New York, NY, USA. 223 pp. DOI: xxx. <a href="links/AR6Chap02.html" target="_blank">Accessible online</a><br>
```

**Review by mauzey1 — CHANGES_REQUESTED** [review]
I would suggest adding the line breaks for the lines I selected.  That would make it look cleaner.

**Comment by durack1** [comment]
@mauzey1 thanks, those changes should be merged into the PR

**Review by mauzey1 — APPROVED** [review]

**Comment by durack1** [comment]
@mauzey1 great, thanks, when is the next cron run to update the external pcmdi.llnl.gov site?

**Comment by mauzey1** [comment]
The site should update at 12:30 PM.

**Comment by durack1** [comment]
Perfect, thanks, it's updated

**Pull request merged**